src/components/Container.js

```diff
@@ -59,28 +61,38 @@ class Container extends React.Component {
     this.state = {
       drawerOpen: false,
       message: null,
+      importExportOpen: false
     };
   }
 
-
   setMessageToNull = () => {
     this.setState({
         message: null
     })
   }
-    newTrailClicked = () => {
-      const { addTrail, trailSelected, selectedTrail } = this.props;
-      let id = new Date().getTime();
-      id = id.toString();
-      const name = 'New Trail';
-      const trail = new Trail({ id, name, features: [] });
-      addTrail(trail);
+
+  setImportExportOpen = (bool) => {
       this.setState({
-        message: 'New Trail Added',
-        drawerOpen: true
+        importExportOpen: bool
       });
     }
 
+  newTrailClicked = () => {
+    const { addTrail, trailSelected, selectedTrail,toggledEditing } = this.props;
+    let id = new Date().getTime();
+    id = id.toString();
+    const name = 'New Trail';
+    const trail = new Trail({ id, name, features: [] });
+    addTrail(trail);
+    trailSelected(selectedTrail, trail.get('id'))
+    toggledEditing(true)
```

Comment:
It would be better if we could just have the reducers respond to the TRAIL_ADDED action instead of calling 3 separate actions.